spec/spec_helper.rb

```diff
@@ -32,6 +32,14 @@
 require 'rails_helper'
 require 'support/spec_test_helper'
 
+# make 5 boards so site_testing doesn't screw up tests
+5.times do
```

Comment:
I can think about cleaner ways to do this but uh regardless there shouldn't be two of these xD